src/components/copy-to-clipboard.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable max-len */
+import React from "react";
+import ReactDOM from "react-dom";
+import Clipboard from "clipboard";
+
+export default class CopyToClipboard extends React.Component {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
You don't need to specify a constructor if all you're doing is calling `super()`